### PR TITLE
Added function and test cases for cross-entropy loss

### PR DESCRIPTION
**Pull request by krishnaw14**
I have added function cross_entropy_loss(X,Y) and test_cross_entropy() for cross-entropy loss that is used as loss function for many binary classification problems.

@norvig if it is alright, should I add more error functions?

### CODE REVIEW

File: tests/test_learning.py [hunk 1]
```diff
@@ -18,6 +18,16 @@ def test_euclidean():
...
+def test_cross_entropy():
+    loss=cross_entropy_loss([1,0], [0.9, 0.3])
+    assert round(loss,2)==0.23
+
+    loss=cross_entropy_loss([1,0,0,1], [0.9,0.3,0.5,0.75])
+    assert round(loss,2)==0.36
+
+    loss=cross_entropy_loss([1,0,0,1,1,0,1,1], [0.9,0.3,0.5,0.75,0.85,0.14,0.93,0.79])
+    assert round(loss,2)==0.26
+
```
Comment (goswami-rahul) [inline tests/test_learning.py]: Add spaces around the `=` and `==`

**Comment by ad71** [comment]
If this isn't in the AIMA book, I don't think we want it in the repository. It is up to @norvig though.

File: tests/test_learning.py (hunk 1 above)
Comment (krishnaw14) [reply tests/test_learning.py]: Done That. Thank you.

**Comment by norvig** [comment]
We have KL divergence, so cross-entropy makes sense.
